src/content/__tests__/monetizationLinkManager.test.ts

```diff
@@ -478,33 +478,65 @@ describe('monetization in main frame', () => {
     ]);
   });
 
-  test('handles rapid attribute changes on monetization link', async () => {
-    const { document } = createTestEnv({
-      head: html`<link rel="monetization" href="${WALLET_ADDRESS[0]}">`,
-    });
-    using linkManager = createMonetizationLinkManager(document);
+  test.failing(
+    'handles rapid attribute changes on monetization link',
+    async () => {
```

Comment:
This should pass I believe.

Reply:
In future then. I saw we don't "collect" quick attribute changes in MonetizationLinkManager